docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -83,76 +78,30 @@ requires 32 Goerli ETH per validator.
     fund the account. You can also request Goerli testnet ETH on the Medalla
     Discord channel.
 
-If you use the [Launchpad] in the next step, then Metamask can handle your
-Ethereum 1.0 account. Otherwise you will need the private key of the account
-when sending the deposit to the deposit contract. The private key can be stored
-in a [password protected V3 Keystore file].
-
-### Generate the validator and send the deposit
+### Generate the validators and send the deposits
 
-!!! tip
+Use the [Medalla Launchpad] to guide you through a step-by-step process to generate your keys and
+send the deposits.
 
-    For small numbers of validators, we recommend using the [Medalla Launchpad]
-    to generate your keys and send the deposits. For more than around 10
-    validators, the workflow below is more convenient.
+!!! note
+    Remember the passwords that you used to create the validator keys, because you need it to [create
+    the validator password files](#create-a-password-file-for-each-validator-key).
 
-    For the Zinken testnet we recommend you use the [Zinken Launchpad] only.
```

Comment:
Do we want to keep a note about Zinken somewhere?  We wouldn't want people to accidentally send deposits to medalla when they meant to send to zinken.

Reply:
Done